index.html

```diff
@@ -12360,7 +12360,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
   &lt;li role="option" aria-setsize="16" <strong>aria-posinset="7"</strong>&gt; cantaloupes &lt;/li&gt;
   &lt;li role="option" aria-setsize="16" <strong>aria-posinset="8"</strong>&gt; dates &lt;/li&gt;
 &lt;/ul&gt;</pre>
-				<p>Authors MUST set the value for <pref>aria-posinset</pref> to an integer greater than or equal to 1, and less than or equal to the size of the set when that size is known. Authors SHOULD use <pref>aria-setsize</pref>.</p>
+				<p>Authors MUST set the value for <pref>aria-posinset</pref> to an integer greater than or equal to 1, and less than or equal to the size of the set when that size is known. If <pref>aria-posinset</pref> is specified, authors MUST specify a value for <pref>aria-setsize</pref>.</p>
 				<p>When exposing <code>aria-posinset</code> on a <rref>menuitem</rref>, <rref>menuitemcheckbox</rref>, or <rref>menuitemradio</rref>, authors SHOULD set the value of <code>aria-posinset</code> with respect to the total number of items in the <rref>menu</rref>, excluding any separators.</p>
```

Comment:
This may be slightly beyond the scope of the issue, but is simple to address. This wording of "exposing" is usually a verb we associate with AT, not authors. I recommend changing to following:

```suggestion
				<p>When specifying <code>aria-posinset</code> on a <rref>menuitem</rref>, <rref>menuitemcheckbox</rref>, or <rref>menuitemradio</rref>, authors SHOULD set the value of <code>aria-posinset</code> with respect to the total number of items in the <rref>menu</rref>, excluding any separators.</p>
```